scripts/mk_win_dist.py

```diff
@@ -299,9 +307,10 @@ def cp_licenses():
     cp_license(False)
 
 def init_flags():
-    global DOTNET_KEY_FILE, JAVA_ENABLED, PYTHON_ENABLED
+    global DOTNET_KEY_FILE, JAVA_ENABLED, PYTHON_ENABLED, ASSEMBLY_VERSION
```

Comment:
Since these globals aren't modified, I think that this `global ...` line is actually wrong in the first place and not needed. But that's separate from these changes.